lib/eleventhframe.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# similar to frame, but for the eleventh frame
+
+class EleventhFrame
+  attr_reader :frame_number, :roll_one, :roll_two
+
+  def initialize(frame_10_spare)
+    @frame_10_spare = frame_10_spare
+    @roll_one = nil
+    @roll_two = nil
+    @strike = false
+    @spare = false
+  end
+
+  def run
```

Comment:
The users rolls in frames 11 and 12 are being printed twice. Believe the score is still being calculated correctly so not a major bug but might confuse the user